lib/CL/pocl_util.h

```diff
@@ -346,6 +346,14 @@ void pocl_str_toupper (char *out, const char *in);
 POCL_EXPORT
 void pocl_str_tolower (char *out, const char *in);
 
+/* Concatenates 'num_str' strings from 'strs' array together into a
+ * new string.
+ *
+ * Returns NULL if num_strs == 0 or allocation failed. Otherwise, return the
```

Comment:
```suggestion
 * @return NULL if num_strs == 0 or allocation failed. Otherwise, return the
```
So Doxygen picks it up.

Reply:
Addressed.